frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -12,7 +17,46 @@ export interface ScheduledEventsTableRowProps {
   start_date: string;
 }
 
+interface RunInstancePayload {
+  start_at: Moment;
+  status: string;
+}
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
+
+const createRunInstance = async (
+  scheduleId: number | undefined,
+  data: RunInstancePayload,
+): Promise<boolean> => {
+  const token = await localForage.getItem<string>(localForageKeys.API_KEY);
+
+  return await axios.request({
+    url: `${BASEPATH}${scheduleId}${api.routes.CREATE_SCHEDULED_INSTANCE}`,
```

Comment:
`scheduleId` can be undefined, but you're continuing to use it without safeguards ... either handle the undefined scenario, or set it to always be defined... I'm for the latter. It should required by this function.

Reply:
Made `scheduleId` required